Sources/GraphIdentifier.swift

```diff
@@ -2,6 +2,14 @@
 //  Copyright © 2019 Swinject Contributors. All rights reserved.
 //
 
+#if os(iOS) || os(macOS)
```

Comment:
Why only on these platforms?

Reply:
Linux didn't recognize `UUID()`. I see a library for `UniqueID()` available but it requires adding a dependency. We could go that route I think.

Reply:
https://forums.swift.org/t/uniqueid-time-ordered-uuids-in-swift/53265

Reply:
Does that mean Linux will still have threading issues?

Reply:
Might be good to call that out at the top of the PR. Do we also want to fix this on `tvOS` and `watchOS`?

Reply:
True! Will do that

Actually, I think can we just leverage random `Int.max` for the Linux case & have that fixed here too?

Reply:
That's probably fine.